portal/templates/redesign/help-and-support_new.html

```diff
@@ -8,13 +8,33 @@
 
 {% block scripts %}
 {{ block.super }}
+<script>
+    $( document ).ready(function() {
+        questions = $('.faqs').find('div').each(function() {
+            $( this ).children('p').html('<span>+　</span>'+$( this ).children('p').html())
+            $( this ).children('p').click(function() {
+                $( this ).parent().children('div').toggle()
+                if ($( this ).children('span').text() == '+　') {
+                    $( this ).children('span').text('–　');
+                }
+                else {
+                    $( this ).children('span').text('+　');
+                }
+            });
+        });
+    });
+
+</script>
```

Comment:
If I recall correctly what @ruthdw has taught me, either now or later we'll want to move this chunk of lonely script and put it in a function in a separate JS file. We would then import that file here, to make things cleaner :)